lib/roken/version-script.mapin

```diff
@@ -219,8 +301,6 @@ HEIMDAL_ROKEN_2.0 {
 		rtbl_set_flags;
 		rtbl_set_prefix;
 		rtbl_set_separator;
-		signal;
```

Comment:
Do we even need the `signal()` wrapper?  And why isn't it called `rk_signal()`?  @jaltman.  Maybe we should just remove it.

Reply:
I'll keep it removed for now then